examples/src/components/AttributeFilterExample.jsx

```diff
@@ -30,11 +25,15 @@ export class AttributeFilterExample extends Component {
     onApply(filter) {
         // eslint-disable-next-line no-console
         console.log("AttributeFilterExample onApply", filter);
-        this.setState({ filters: [], error: null });
-        if (filter.in) {
-            this.filterPositiveAttribute(filter);
+        const filterType = filter.positiveAttributeFilter
+            ? "positiveAttributeFilter"
+            : "negativeAttributeFilter";
+        const inType = filterType === "positiveAttributeFilter" ? "in" : "notIn";
```

Comment:
const inType = AFM.isPositiveAttributeFilter(filter) ? "in" : "notIn";

Reply:
This was it's forward compatible with typescript

```const filterItems = AFM.isPositiveAttributeFilter(filter) ? filter.positiveAttributeFilter[inType] : filter.negativeAttributeFilter[inType];```

Reply:
Thx, updated.